### PR TITLE
VCluster API Refresh

### PR DESCRIPTION
**Pull request by chinhtranvan**
This picks up the latest vclusterops package. The key change is it will hide sensitive config parms from the various logs. Originally, it was to use a new parameter to pass down config parms to create DB, but after a few revisions we opted to reuse the existing field, so this just becomes a refresh of the package and no API changes.

### CODE REVIEW

File: pkg/vadmin/create_db_vc.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ func (v *VClusterOps) genCreateDBOptions(s *createdb.Parms, certs *HTTPSCerts) v
 	}
 
 	// Additional configuration parameters for create db.
-	opts.ConfigurationParameters = s.ConfigurationParams
```
Comment (spilchen) [inline pkg/vadmin/create_db_vc.go:78]: I'm trying to understand the vclusterops changes again. We still have this parameter (`ConfigurationParameters`). Why would you want to use `ConfigurationParameters` vs `CommunalStorageParameters`? They both seem to do the same thing. In fact, we combine the maps when we call the bootstrap catalog op. We have masking for sensitive parms, but it doesn't seem like it matters where it comes from.

Comment (chinhtranvan) [reply pkg/vadmin/create_db_vc.go:78]: I agree that they both seem to do the same thing. AT is using both ConfigurationParameters and CommunalStorageParameters, and our vcluter mimic AT, there is some reasons e.g. For usability -- users understand better on the effect of the parameters they pass in

Comment (spilchen) [reply pkg/vadmin/create_db_vc.go:78]: Thanks Chinh

**Comment by chinhtranvan** [comment]
@spilchen I just sync the latest vcluster repo

**Pull request merged**